.github/workflows/dummy_lambda.yml

```diff
@@ -1,12 +1,39 @@
 name: deploy-dummy-lambda
 on:
   push:
-    branches:
-      - lambda-deploy-action
+    branches: master
     paths:
       - 'dummy_lambda/**'
+  pull_request:
+    branches: master
 jobs:
+  Test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Python 3
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r tests/requirements.txt
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.TEST_AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.TEST_AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ secrets.TEST_SKYPE_SENDER_REGION }}
+      # Execute test for daily message lambda
+      - name: Run test for daily message lambda
+        run: coverage run -m pytest tests/test_integration_daily_messages_lambda.py
+      # Execute Test for skype lambda
+      - name: Run test for skype lambda
+        run : coverage run -m pytest tests/test_integration_skype_lambda.py
```

Comment:
I was just going over the coverage tool. I see this option:
  --fail-under=MIN      Exit with a status of 2 if the total coverage is less

We are running with coverage, but we are not checking the coverage percentage. Should we include a .coveragerc with more configuration on when to fail the test? or maybe we can use the above argument (--fail-under) and set a standard coverage percent below which the tests should fail.

Reply:
Currently there is defect where coverage report shows coverage as 0%. This is with `coverage 5.3 and 5.3.1`. We can definitely think of adding coverage percentage check once this issue is fixed in later coverage version.

https://github.com/nedbat/coveragepy/issues/1034

If you run `coverage report tests/<<testfilename.py>> or `coverage report -i tests/test_integration_skype_lambda.py --fail-under=80` then you can see coverage showing as 0.

Reply:
@rahul-bhave, the issue you quoted doesn't seem to be related, because for the Author of that issue, when he runs the test file individually, he is getting the coverage as 100% and when he runs it on the folder, its 0%. Also, if you look at one of his comments, he mentioned that the issue is because he seems to be using an incompatible library 'import ptvsd". I dont think we are using that library, also, as you mentioned, we are getting 0% even if we run on an individual file.